content/sql/concepts/wildcards/wildcards.md

```diff
@@ -0,0 +1,55 @@
+---
+Title: Wildcards
+Description: 'Wildcards are special characters that are used in SQL to represent one or more arbitrary characters.'
+Subjects:
+  - 'Data Science'
+  - 'Computer Science'
+Tags:
+  - 'Operators'
+  - 'Database'
+  - 'Queries'
+  - 'SQLite'
+  - 'MySQL'
+  - 'PostgreSQL'
+CatalogContent:
+  - 'learn-sql'
+  - 'paths/analyze-data-with-sql'
+  - 'paths/design-databases-with-postgresql'
+---
+
+Wildcards are special characters that are used in SQL to represent one or more arbitrary characters. They allow flexibility in matching string values using the LIKE operator.
+
+There are two wildcards generally recognized across SQL implementations:
+
+- `%` which matches any arbitrary number of characters. (Including 0 characters.)
+- `_` which matches any one single character.
+
+Some SQL implementations add additional types of wildcards. (Microsoft Access SQL, in particular, is non-standard, using `*` in place of `%` and `?` in place of `_`.)
+
+Wildcards are typically used with the [`LIKE`](https://www.codecademy.com/resources/docs/sql/operators/like) operator, which returns true if the first string argument matches the wildcard pattern in the second.
+
+## Examples
+
+Select all rows where `column`'s value has "H" as the second character:
```

Comment:
```suggestion
The following example selects all rows where "H" is the second character in the value of `column`:
```

I'm personally curious, what is the `%` doing in this example? It's checking for zero or more instances of such a `column` value?

Reply:
It's matching the rest of the string after the 'H'. Without it, it would only match a two character string ending with 'H'.

Reply:
Oh ok gotcha. 🙏